server/views/partials/vote.pug

```diff
@@ -0,0 +1,11 @@
+td
+  
+  .form-check.btn-group.btn-group-toggle
+    label.btn.btn-primary(v-text="(votes.indexOf(team._id) !== -1 ? 'selected' : '')", v-bind:class="{ active: votes.indexOf(team._id) !== -1 }")
+    input(
+      type="checkbox",
+      :id="'vote_'+team._id",
+      v-model="votes",
+      :value="team._id",
+      v-on:click="validateVote( (votes.indexOf(team._id) !== -1 ? team._id : encodeURIComponent(JSON.stringify(votes))), (votes.indexOf(team._id) === -1), $event)",
```

Comment:
here -- if the current view's votes variable contains this team's teamId, then this is an un-click action, and only a single teamId is sent as a string. If else, it should go over as an encoded array to the server.

Reply:
Still a problem -- error in browser console: `votes.indexOf is not a function`